src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/Comments.component.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import type { ApiTEIEvent } from 'capture-core/events/getEnrollmentEvents';
+import React, { type ComponentType } from 'react';
+import { IconMessages16 } from '@dhis2/ui';
+import { withStyles } from '@material-ui/core';
+
+type Props = {|
+    event: ApiTEIEvent,
+    ...CssClasses,
+|};
+
+const styles = {
+    wrapper: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+    text: {
+        paddingLeft: '2px',
+    },
+};
+export const CommentsPlain = ({ event, classes }: Props) => {
+    const nbComments = event.notes?.length;
```

Comment:
Hey,
Does the first part `nb` of the`nbComments` have any meaning? 
Thanks!

Reply:
it's number of comments

Reply:
I think a good rule of thumb is: prefer writing the entire word(s) unless the abbreviation is more readable and immediately understandable to other developers. 

Suggestion for this particular variable name: `commentsCount`

In general, we should aim for descriptive variable names to improve readability and maintainability.

Reply:
in this case I think it's quite obvious that it points to the total of comments

Reply:
Let's put this on the agenda for our meeting next week 🙂